whatToDad_site/whatToDad_app/templates/activity.html

```diff
@@ -26,9 +26,52 @@ <h1> Activities </h1>
     <br>
     <form method="POST" action="{% url 'activity' %}">
         {% csrf_token %}
-        {{ form }}
-        <input type="button" class="btn btn-secondary" value="Cancel" data-dismiss="modal"></input>
-        <input type="submit" class="btn btn-primary" name="add" value="Add activity"></input>
+        <div class="row">
+            <div class="col">
+                <h3 class="form-control">Detail: {{ form.detail }}</h3>
+            </div>
+
+            <div class="col">
+                <h3 class="form-control">Min-age: {{ form.min_age }}</h3>
```

Comment:
There's an input type `range` that can be used to get a slider. With that you can control quite a lot and make it a little easier to adjust the number, display tick marks (with labels) where you think there's meaningful ages, like 0, 3, 8, 12, 18, 21, 30, 65, etc. This would help users understand your intention with the date ranges better. 

Lots of good info here, especially towards the end, on how it can be used to encompass a good amount of data and help users understand
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range